enr/enr.go

```diff
@@ -28,6 +31,40 @@ type ENR struct {
 	Udp6Port uint16   // IPv6-specific UDP port. If omitted, same as UdpPort.
 }
 
+// Returns the address of the node, which is the keccak256 hash of its Secp256k1 key.
+func (enr ENR) NodeAddr() []byte {
+	return keccak(enr.Secp256k1)
+}
+
+// Returns the address of the node as a hex encoded string.
+func (enr ENR) NodeAddrHex() string {
+	return fmt.Sprintf("%x", enr.NodeAddr())
+}
+
+// computes the distance between two ENRs defined as 
+// log_2 (keccak256(n1) XOR keccak256(n2))
+func LogDistance(n1, n2 *ENR) int {
```

Comment:
Since this is used mostly in the context of the k table, can we move this function into that package?